src/docs/content/observability/observability-scenario/pre-requisites/setup.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 1
+tags:
+  - mushop setup
+---
+
+## Pre-requisites
+
+### Install Ingress Controller
+
+```bash
+kubectl create namespace mushop
+```
+```text
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/provider/cloud-generic.yaml
+```
+
+### Create a Secret for OCI Monitoring
+
+```bash
+kubectl create secret generic monitoring-secret \
```

Comment:
Include OCI credentials should never be something mandatory. Also, there's a lot of intelligence on the scripts to reuse ocicredentials secrets. Create a separate secret for a specific service should be a second optional option.

Reply:
It is an optional step. I will correct doc.